website/events/migrations/0068_eventdocument.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 4.1.7 on 2023-03-22 19:58
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def populate_eventdocument(apps, schema_editor):
+    EventDocument = apps.get_model("documents", "EventDocument")
+    NewEventDocument = apps.get_model("evetns", "EventDocument")
```

Comment:
```suggestion
    NewEventDocument = apps.get_model("events", "EventDocument")
```
I'm guessing this still needs to be tested (locally)